src/main/java/at/hannibal2/skyhanni/features/event/hoppity/HoppityEggsCompactChat.kt

```diff
@@ -65,17 +65,19 @@ object HoppityEggsCompactChat {
     private fun createCompactMessage(): String {
         val mealName = lastChatMeal?.coloredName ?: ""
         val mealNameFormatted = if (rabbitBought) "§aBought Rabbit" else "$mealName Egg"
+        val showRarity = HoppityEggsManager.config.rarityInCompact
 
+        val rarityText = if (showRarity) "$lastRarity " else ""
         return if (duplicate) {
             val format = lastDuplicateAmount?.shortFormat() ?: "?"
             val timeFormatted = lastDuplicateAmount?.let {
                 ChocolateFactoryAPI.timeUntilNeed(it).format(maxUnits = 2)
             } ?: "?"
 
             val timeStr = if (config.showDuplicateTime) ", §a+§b$timeFormatted§7" else ""
-            "$mealNameFormatted! §7Duplicate $lastName §7(§6+$format Chocolate§7$timeStr)"
+            "$mealNameFormatted! §7Duplicate $rarityText$lastName §7(§6+$format Chocolate§7$timeStr)"
```

Comment:
should never show rarity for duplicate rabbits, or having a separate option

Reply:
Why wouldn't we? If they're turning on the option, wouldn't it make sense for consistency? Why different options for new versus duplicate?

Reply:
I don't care about duplicates nearly as much as I care about uniques. For dupes, the current format with just showing the rarity as color for the rabbit name is perfectly fine for me. Having a bold COMMON text next to it would annoy me, I maybe even think this is unique/special.
![image](https://github.com/user-attachments/assets/6ab8ed2f-5a81-45f6-a2cb-8d9d08a5beae)

Instead, for truly unique rabbits, having the rarity blast next to the name sounds like a great idea.

This is just my personal preference about this. Therefore, I think we can add those as settings for the config.

Or simply not show rarity name for dupes always. The dupe message is intentionally designed to be less shiny/flashy as the purple big NEW text for non-dupes.